src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -97,7 +97,6 @@ class RoutedSavableReviewPage extends React.Component {
             formConfig={formConfig}
             pageList={pageList}
             path={path}
-            renderErrorMessage
```

Comment:
Since `RoutedSavableReviewPage` is the only place where `SubmitController` gets used in current production code, this `renderErrorMessage` prop can be removed as it has no actual effect.